src/settings/MappingProfiles/TransformationsList/TransformationsList.js

```diff
@@ -39,14 +45,22 @@ export const TransformationsList = ({
     transformation: intl.formatMessage({ id: 'ui-data-export.mappingProfiles.transformations.transformation' }),
   }), [intl]);
 
+  useEffect(() => {
+    const formattedTransformations = orderBy(transformations.map(transformation => ({
+      fieldName: formatter.fieldName(transformation),
+      transformation: formatter.transformation(transformation),
+    })), 'fieldName', 'asc');
+
+    setSortedTransformations(formattedTransformations);
+  }, [transformations, formatter]);
+
   return (
     <MultiColumnList
       id="mapping-profile-transformations-list"
-      contentData={transformations}
+      contentData={sortedTransformations}
       columnMapping={columnMapping}
       columnWidths={columnWidths}
       visibleColumns={visibleColumns}
-      formatter={formatter}
```

Comment:
Is formatter no longer needed?

Reply:
It's not needed as MLC property, since I formatted the data inside this component in order to sort it.